linkerd/announcer/serversets/src/main/scala/com/twitter/finagle/zookeeper/buoyant/ZkAnnouncer.scala

```diff
@@ -0,0 +1,15 @@
+package com.twitter.finagle.zookeeper.buoyant
+
+import com.twitter.finagle.{Announcement, Announcer}
+import com.twitter.finagle.zookeeper.{ZkAnnouncer => FZkAnnouncer, DefaultZkClientFactory}
+import com.twitter.util.Future
+import java.net.InetSocketAddress
+
+class ZkAnnouncer(hosts: String, pathPrefix: String) extends Announcer {
+  override val scheme: String = "zk-serversets"
+
+  val underlying = new FZkAnnouncer(DefaultZkClientFactory)
+
+  override def announce(addr: InetSocketAddress, name: String): Future[Announcement] =
+    underlying.announce(addr, s"$hosts!$pathPrefix/$name!0")
```

Comment:
I think we have a choice about how we want to handle names that contain slashes

1. require that names start with / and treat them as paths
2. require that names don't start with / and treat them as paths
3. escape/encode slashes